src/components/ChannelInner.js

```diff
@@ -693,14 +690,7 @@ class ChannelInner extends PureComponent {
           enabled={!this.props.disableKeyboardCompatibleView}
         >
           <ChannelContext.Provider value={this.getContext()}>
-            <SuggestionsProvider
-              handleKeyboardAvoidingViewEnabled={(trueOrFalse) => {
```

Comment:
don't need this keyboard avoiding view enabler anymore?

Reply:
nah it wasn't doing anything